roles/create-openshift-resources/tasks/create_app.yml

```diff
@@ -88,8 +103,8 @@
   delay: 5
   ignore_errors: yes
 
-## This is confusing to me. Dunno why the app_item is not passed to the includes context...
-- name: Set Facts Not Sure Why
+# http://docs.ansible.com/ansible/playbooks_loops.html#loops-and-includes-in-2-0
+- name: Set Facts
```

Comment:
Fixed. Please do note the comment with link to documentation above the task as well